packages/components/src/components/contract-card/contract-card-items/tick-counter-bar.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { localize } from '@deriv/translations';
+import Text from '../../text';
+
+const TickCounterBar = ({ current_tick, max_ticks_duration }) => (
+    <div className='dc-tick-counter-bar__container'>
+        <div className='dc-tick-counter-bar__track'>
+            <Text size='xxs' weight='bold' className='dc-tick-counter-bar__text'>
+                {localize('{{current_tick}}/{{max_ticks_duration}} Ticks', {
```

Comment:
@maryia-binary We should not be using localize in the components package.
We don't extract translation content from the components package. Please check `extract-string.js` file